### PR TITLE
Fix a memory crash (can't create more wxBitmap) when tiles are re-generated if the tileset contains a lot of tiles

### PR DESCRIPTION
**Pull request by victorlevasseur**
To display the tiles inside the tilemap editor, the extension was creating a wxBitmap for each tiles when the TileSet::Generate() method was called. It appears that it can cause crash when there are a lot of tiles (wxBitmap functions failling without any visible problem).

To solve this issue, there is now only one wxBitmap containing the whole tileset. In the editors, the wxDC::Blit method is used (with a wxMemoryDC containing the tileset wxBitmap) to display tiles instead of individual wxBitmaps.

(Need testing)

### CODE REVIEW

File: Extensions/TileMapObject/TileSet.cpp [hunk 1]
```diff
@@ -13,6 +13,7 @@ This project is released under the MIT License.
 #include <wx/file.h>
 #include <wx/filefn.h>
 #include <wx/filename.h>
+#include <wx/image.h>
```
Comment (4ian) [inline Extensions/TileMapObject/TileSet.cpp]: I didn't tried, but as you only removed code, is this include necessary? :)

File: Extensions/TileMapObject/TileMapPanel.cpp [hunk 2]
```diff
@@ -139,7 +143,16 @@ void TileMapPanel::OnPaint(wxPaintEvent& event)
...
+                dc.Blit(GetPositionOfTile(col, row).x, 
+                        GetPositionOfTile(col, row).y,
+                        m_tileset->tileSize.x,
+                        m_tileset->tileSize.y,
+                        &tilesetDC,
+                        m_tileset->GetTileTextureCoords(m_tilemap->GetTile(layer, col, row)).topLeft.x,
+                        m_tileset->GetTileTextureCoords(m_tilemap->GetTile(layer, col, row)).topLeft.y,
+                        wxCOPY,
+                        true);
+                //dc.DrawBitmap(m_tileset->GetTileBitmap(m_tilemap->GetTile(layer, col, row)), GetPositionOfTile(col, row).x, GetPositionOfTile(col, row).y, true);
```
Comment (4ian) [inline Extensions/TileMapObject/TileMapPanel.cpp]: You should delete this line :) If Blit works properly, there is not reason to keep this line and if we need to go back to the previous method for whatever reason we can just inspect the git ;)

**Comment by 4ian** [comment]
Sounds a way better method indeed! Thanks for noticing it and fixing it! :)
I've made some comments on the code because I can test it for now.. If it works properly after doing some testing on your side it should be ok to merge it! :)

**Comment by victorlevasseur** [comment]
Can be merged.

**Comment by 4ian** [comment]
:+1:

**Pull request merged**